.circleci/config.yml

```diff
@@ -44,8 +44,8 @@ commands:
     steps:
       - restore_cache:
           keys:
-            - v1-gems-{{ checksum "~/amplify-ios/Gemfile.lock" }}
-            - v1-gems-
+            - v2-gems-{{ checksum "~/amplify-ios/Gemfile.lock" }}
+            - v2-gems-
```

Comment:
is the `v2-gems-` key provided by CircleCI? If not, where is the content for that cache saved?

Reply:
I invalidated the cache by bumping it to v2. It's saved here: https://github.com/aws-amplify/amplify-ios/pull/610/files#diff-1d37e48f9ceff6d8030570cd36286a61R99

Reply:
My read of that file reference is that it sets a cache with a key comprising both the static `v2-gems-` string and the interpolated checksum. I'm wondering about L48, which specifies just the static string `v2-gems-`. It may be innocuous; if so, please disregard

Reply:
Ah, I see. That's a cache key strategy suggested by CircleCI where it will try to match on the full key first but then falls back to the partial match

Reply:
So in our case, if it's a partial match, it'll pull the previous gems and then perform a bundle install after. The logic is that it'll be a tad quicker to do that than a complete fresh install